packages/edit-site/src/components/add-new-pattern/index.js

```diff
@@ -168,8 +168,7 @@ export default function AddNewPattern() {
 							params.categoryId !== 'my-patterns'
 						) {
 							history.push( {
-								path: `/patterns`,
-								categoryType: PATTERN_TYPES.theme,
```

Comment:
For the distinction between patterns and template parts, I just used the postType and remove the categoryType from the url.

Reply:
This is fixed.